README.md

```diff
@@ -42,20 +42,20 @@ On all platforms run the following in the VireoSDK folder to install JavaScript-
 - emsdk toolchain (see following)
 
 ### Installing the emsdk toolchain
-The emsdk toolchain is used to cross-compile Vireo to Vireo.js to run in Node.js and Browser environments.
+emscripten (emsdk) is used to cross-compile Vireo to Vireo.js to run in Node.js and web browser environments.
 
-1. Create a directory outside of the VireoSDK folder to store the emsdk toolchain binaries
+1. Create a directory outside of the VireoSDK folder to store the emsdk toolchain binaries.
 
 2. Do one of the following:
```

Comment:
If you want to remove the or then emphasize the one. `**ONE**`

Reply:
Good idea.